examples/create-setters/simple/README.md

```diff
@@ -0,0 +1,94 @@
+# create-setters: Simple Example
```

Comment:
There's some inconsistent use of the noun "setter". Does it refer to the setter reference or the parameter itself. 

apply-setter -> Apply the values of the parameters to reference site
create-setter -> Create references

This is better named as create-setter-ref or create-setter-comments or something like that.

Reply:
Actually, nvm. `create-setters` is how 0.39 CLI used it, so it's better to keep it as is.